src/resources/i18n/translations/en.json

```diff
@@ -179,28 +179,28 @@
 	"ARIA HINT - unsubscribe from": "unsubscribe from",
 	"ARIA LABEL - Queue Last": "Queue Last",
 	"ARIA LABEL - Queue Next": "Enqueue as next",
-	"About Premium": "About Podverse premium",
+	"About Premium": "About Podverse Premium",
 	"About dynamic ads": "About dynamic ads",
 	"About": "About",
 	"Account": "Account",
-	"Active Downloads": "Active downloads",
+	"Active Downloads": "Active Downloads",
 	"Actual amount will be higher due to network fees": "The actual amount also include network fees",
-	"Add Custom RSS Feed": "Add custom RSS feed",
-	"Add Wallet": "Add wallet",
-	"Add current item next in queue when playing a new item": "Enque current item as next when playing something",
-	"Add to Playlist": "Add to playlist",
+	"Add Custom RSS Feed": "Add Custom RSS Feed",
+	"Add Wallet": "Add Wallet",
+	"Add current item next in queue when playing a new item": "Add current item next in queue when playing a new episode",
```

Comment:
```suggestion
	"Add current item next in queue when playing a new item": "Skip the queue to start playback",
```

Reply:
This doesn't make sense to me either. I don't think it describes the function the setting enables. The queue isn't getting "skipped". You're just playing a new item, and the current item gets moved into the queue when you play the new item.

Reply:
Yeah I don't have it. The current version works for now, but I know there are improvements to be had.